core/src/main/java/org/elasticsearch/index/fielddata/plain/IndexIndexFieldData.java

```diff
@@ -125,6 +131,21 @@ public AtomicOrdinalsFieldData loadDirect(LeafReaderContext context)
     }
 
     @Override
+    public SortField sortField(@Nullable Object missingValue, MultiValueMode sortMode, XFieldComparatorSource.Nested nested, boolean reverse) {
+        XFieldComparatorSource source = new BytesRefFieldComparatorSource(this, missingValue, sortMode, nested);
+        if (nested != null ||
+            (sortMode != MultiValueMode.MAX && sortMode != MultiValueMode.MIN) ||
+            (source.sortMissingFirst(missingValue) == false && source.sortMissingLast(missingValue) == false)) {
+            return new SortField(getFieldName(), source, reverse);
```

Comment:
can you indent the content of the if statement and the content of the block at different levels, otherwise I find it a bit hard to read